### PR TITLE
SSO API QE structure

### PR DESCRIPTION
**Pull request by PacoLuna-AT**

### CODE REVIEW

File: src/test/resources/config.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+mongodb+srv://at-sso-api:4gil3th0ught@cluster0.vliih.mongodb.net/at-sso-db?retryWrites=true&w=majority
```
Comment (at-internship) [inline src/test/resources/config.properties]: We don't need this sensitive info here, if needed, please don't commit it

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/test/resources/config.properties (hunk 1 above)
Comment (PacoLuna-AT) [reply src/test/resources/config.properties]: Okey, I will put the file on the gitignore

**Review by at-internship — APPROVED** [review]
Approved

**Review by AdanMerino-AT — APPROVED** [review]

**Comment by DRuizAT** [comment]
Approved

**Pull request merged**